impl/src/main/java/com/okta/sdk/impl/resource/DefaultPolicyRuleBuilder.java

```diff
@@ -25,20 +24,12 @@
 
 public class DefaultPolicyRuleBuilder<T extends PolicyRuleBuilder> implements PolicyRuleBuilder<T> {
 
-    protected String id;
```

Comment:
Was this removed from the spec?

Reply:
yes, its a `read-only` field now. So, no need if this in the builder class.